demo/launch/pipeline_testbench.launch.py

```diff
@@ -98,6 +98,31 @@ def launch_setup(context, *args, **kwargs):
         }
     }
 
+    drake_ktopt_planning_pipeline_config = {
+        "drake": {
+            "planning_plugins": [
+                "ktopt_interface/KTOptPlanner",
+            ],
+            "request_adapters": [
+                "default_planning_request_adapters/ResolveConstraintFrames",
+                "default_planning_request_adapters/ValidateWorkspaceBounds",
+                "default_planning_request_adapters/CheckStartStateBounds",
+                "default_planning_request_adapters/CheckStartStateCollision",
+            ],
+            "response_adapters": [
+                "default_planning_response_adapters/AddTimeOptimalParameterization",
```

Comment:
Yeah, I just left everything as is from the STOMP implementation just in case, I'll do a pass of optimizations and remove everything that is not required

Reply:
Yup, will do a pass of optimizations to remove unnecessary options. For now creating an issues